Xamarin.Forms.Core/ScrollView.cs

```diff
@@ -26,6 +26,10 @@ public class ScrollView : Layout, IScrollViewController, IElementConfiguration<S
 
 		readonly Lazy<PlatformConfigurationRegistry<ScrollView>> _platformConfigurationRegistry;
 
+		public static readonly BindableProperty HorizontalScrollBarVisibilityProperty = BindableProperty.Create("HorizontalScrollBarVisibility", typeof(ScrollBarVisibility), typeof(ScrollView), ScrollBarVisibility.Default);
```

Comment:
Consider using `nameof(HorizontalScrollBarVisibility)` instead of a string here to make refactoring cleaner if needed. Also helps avoiding typos or at least makes sure all typos are the same :P

Reply:
I normally do in my own code, I'm glad to change it, I was trying to keep the style similar to others I saw in the files for consistency. @jassmith   Would you like me to update them all to use nameof while I'm at it? I thought about it, but didn't want to change items that were not related.

Reply:
nameof wasn't always there in the old days :) All new properties should use it. I have no problem if you want to submit a separate PR to fix all of them :P